client/app/bundles/course/video/submission/containers/VideoControls/PlayButton.jsx

```diff
@@ -14,13 +15,17 @@ const propTypes = {
 };
 
 const PlayButton = (props) => {
-  const playIconClass = props.playing ? 'fa fa-pause' : 'fa fa-play';
+  const playIconClass = props.playing ? (
+    <Pause fontSize="inherit" />
+  ) : (
+    <PlayArrow fontSize="inherit" />
+  );
   return (
     <span
       className={styles.playButton}
       onClick={() => props.onClick(props.playing)}
     >
-      <i className={playIconClass} />
+      {playIconClass}
```

Comment:
`buttonIcon`
Please be more mindful with the variable naming.

Reply:
okay noted. I'm sorry I was not really giving much thought with the naming (I was just focusing on replacing the icons but forgot entirely about the naming) will be more careful next time :)